DEVELOPERS.md

```diff
@@ -68,6 +68,17 @@
          `Console` tab to view printed logs and errors, the `Element` tab to
          inspect the HTML and CSS, the `Network` tab for network calls, etc).
 
+#### Debugging Add-in: Method 2
+If the above method to debug the add-in is not working, one can try the following steps:
+   1. Delete the `C:\Users\{username}\.office-addin-dev-certs` folder.
+   2. Run `npx office-addin-dev-certs install`.
+   3. Now, run `npm run start` to startup the server and sideload the add-in.
+   4. Click on Add-in to open it.
+   5. Instead of attaching the debugger from add-in, open VS Code and click on 
+      `Run and Debug`(Shortcut key on Windows:Ctrl+Shift+D). Click on the green
```

Comment:
- Enclose the keyboard keys in `<kbd>` tags.
- S in shortcut won't be in capital.
- Space before `(` and after `:`.

Reply:
@algomaster99 
Made the changes.